libraries/chain/steem_evaluator.cpp

```diff
@@ -532,6 +532,12 @@ namespace golos { namespace chain {
                 }
 
                 GOLOS_CHECK_PARAM(percent, {
+                    if (!_db.has_hardfork(STEEMIT_HARDFORK_0_21__1009)) {
+                        GOLOS_CHECK_VALUE(STEEMIT_MIN_CURATION_PERCENT_PRE_HF21 <= percent && percent <= STEEMIT_MAX_CURATION_PERCENT,
+                            "Curation rewards percent must be between ${min} and ${max}.",
+                            ("min", STEEMIT_MIN_CURATION_PERCENT)("max", STEEMIT_MAX_CURATION_PERCENT));
+                    }
```

Comment:
This checking shouldn't happen.
The checking of mprops should work.

Reply:
@afalaleev Look at this checking:
https://github.com/GolosChain/golos/blob/1009-min-curation-percent-remove-witness-limit/libraries/protocol/steem_operations.cpp#L170

In pre-HF21 it was:
25% <= percent
And it was failing independent from mprops.
And it will continue failing on these nodes with old version.
In new nodes it should fail, too, if no HF21.

About default value you are right.